demos/default/pages/css/index.tsx

```diff
@@ -12,23 +12,27 @@ function BuiltInCSSSupport() {
   return (
     <div>
       <h1>Built In CSS Support</h1>
-      <Link href="https://nextjs.org/docs/basic-features/built-in-css-support"><a>Read Docs</a></Link>
```

Comment:
I mentioned it in the other PR, but dropping it here as well for context to this change. See https://github.com/netlify/next-runtime/pull/1714#discussion_r1004653174